go.mod

```diff
@@ -12,11 +12,14 @@ require (
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pborman/uuid v0.0.0-20160209185913-a97ce2ca70fa
 	github.com/robfig/cron v1.2.0
+	github.com/shirou/gopsutil v3.21.11+incompatible
```

Comment:
They said 3.21.11 https://github.com/shirou/gopsutil/releases/tag/v3.22.12  has some issues so they reverted it. Could you verify that?